lib/imlib/simd.h

```diff
@@ -1290,6 +1345,74 @@ static inline uint32_t vmladava_u16(v128_t v0, v128_t v1, uint32_t acc) {
     #endif
 }
 
+static inline v4x_rows_t vcvt_f32_u8(v128_t v0) {
```

Comment:
```suggestion
static inline v4x_rows_t vcvt_u8_f32(v128_t v0) {
```
Same for the others.

Reply:
Updated

Reply:
@kwagyeman I missed this. Just re-request preview in the future.